specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json

```diff
@@ -2752,7 +2784,8 @@
         },
         "order": {
           "description": "The order in which the rules are applied for the endpoint. Possible values {0,1,2,3,………}. A rule with a lesser order will be applied before a rule with a greater order. Rule with order 0 is a special rule. It does not require any condition and actions listed in it will always be applied.",
-          "type": "integer"
+          "type": "integer",
+          "format": "int32"
```

Comment:
Instance of change format. Usually only a potential break (minor).

Reply:
The validation required that this property have a format. Since 2021-06-01 is a completely new API version we are not causing breaking changes.